src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```diff
@@ -43,6 +43,10 @@
 
       <Content Remove="@(_ContentToUpdate)" />
       <Content Include="@(_ContentToUpdate)" ExcludeFromContentCheck="true" PublishFolderType="None" />
+
+      <!-- Remove other extraneous files -->
+      <Content Remove="global.json" />
+      <Content Remove="Properties\launchSettings.json" />
```

Comment:
I saw these targets bringing in these files:

https://github.com/dotnet/sdk/blob/ee1eafa92eb71eb9cc53cc4866b57ce9d8818b00/src/RazorSdk/Sdk/Sdk.Razor.StaticAssets.ProjectSystem.props#L33

Is this the right thing to do here? /cc @pranavkm @Eilon 

We had a workaround in the `maui-blazor` template for `launchSettings.json`, would be nice to take that out.

Reply:
One thing we saw with this is that it removed it from the IDE so there was no way to edit this file.

May e we need to add it back as none?

Reply:
Hmm I don't recall the history here but it's worth trying what @mattleibow suggests.

Adding @javiercn because this might be related to Static Web Assets and some of the trickiness with that.

Reply:
It's a purely dev-artifact. Nothing outside of `dotnet-run` / VS is meant to read it. I think it got accidentally categorized as `Content` because the **.json glob in the RazorSDK is too aggresive.

Reply:
@mattleibow the changes here removes these files from `@(Content)` during the `ConvertStaticWebAssetsToMauiAssets` target, so I don't think it changes anything in the IDE behavior?

They'll be in `@(Content)` when VS evaulates the project.